stream_alert/rule_processor/sink.py

```diff
@@ -32,14 +36,13 @@ def __init__(self, env):
             env (dict): loaded dictionary containing environment information
         """
         self.env = env
+        self.client_dynamo = boto3.client('dynamodb', region_name=self.env['lambda_region'])
         self.client_lambda = boto3.client('lambda', region_name=self.env['lambda_region'])
         self.function = os.environ['ALERT_PROCESSOR']
+        self.table = os.environ['ALERT_TABLE']
 
-    def sink(self, alerts):
-        """Sink triggered alerts from the StreamRules engine.
-
-        Args:
-            alerts (list): a list of dictionaries representating json alerts
+    def _sink_lambda(self, alerts):
```

Comment:
We should have just done away with this `sink` moniker now 😂

Reply:
+1, let's move away from `sink` altogether in classes/methods/filenames